<?xml version="1.0"?>
<launch>
  <arg name="device" default="/dev/ttyUSB0"/>
	<node pkg="arduimu_v3" type="arduimu_v3" name="arduimu_v3" output="screen">
		<param name="frame_id" value="/arduimu_v3"/>
		<param name="filter_imu" value="true"/>
		<param name="use_mag" value="false"/>
		<param name="device" value="$(arg device)"/>
	</node>
</launch>
